misc/keyvaluestorage/src/node-js/level_db.ts

```diff
@@ -0,0 +1,22 @@
+let database
+
+export default {
+  create(db: string) {
+    if(!database) {
+      if(db == ':memory:') {
+        const { MemoryLevel } = require('memory-level')
+        database = new MemoryLevel({encodingType: 'json'})
+      } else { 
+        try { 
```

Comment:
I think this try needs to be above i.e. to encapsulate `require('memory-level')` require as well?